### PR TITLE
[docs] Add "Getting Started" article

### PR DESCRIPTION
**Pull request by valfirst**

### CODE REVIEW

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+. Build the project:
++
+- macOS / Linux:
++
+[source,shell]
+----
+./gradlew build
+----
+
+- Window:
```
Comment (ikalinin1) [inline docs/modules/ROOT/pages/getting-started.adoc:52]:
```suggestion
- Windows:
```

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+. Generate Eclipse configuration files:
++
+- macOS / Linux:
++
+[source,shell]
+----
+./gradlew eclipse
+----
+
+- Window:
```
Comment (ikalinin1) [inline docs/modules/ROOT/pages/getting-started.adoc:77]:
```suggestion
- Windows:
```

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+. Execute your tests via Gradle:
++
+- macOS / Linux:
++
+[source,shell]
+----
+./gradlew runStories
+----
+
+- Window:
```
Comment (ikalinin1) [inline docs/modules/ROOT/pages/getting-started.adoc:97]:
```suggestion
- Windows:
```

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+=== Install and configure Git
+. Download and install http://git-scm.com/downloads[Git client]
+. https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity[Configure] your identity
+
+=== Install Java
+. https://jdk.java.net/17/[LTS JDK 17] is the baseline. https://jdk.java.net/11/[LTS JDK 11] and the latest https://jdk.java.net/18/[JDK 18] are supported as well.
+. https://www.baeldung.com/java-home-on-windows-7-8-10-mac-os-x-linux[Configure `JAVA_HOME` environment variable] to point your directory with installed JDK.
+. Add new entry to `PATH` environment variable:
+    * for Windows: `%JAVA_HOME%/bin`
+    * fo macOS / Linux: `$JAVA_HOME/bin`
```
Comment (ikalinin1) [inline docs/modules/ROOT/pages/getting-started.adoc:21]:
```suggestion
    * for macOS / Linux: `$JAVA_HOME/bin`
```

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+== Before you start
+In order to use VIVIDUS effectively, you need to know what it is and understand
+some of its fundamental concepts. So before you start using VIVIDUS in earnest,
+we highly recommend you read xref:ROOT:tests-configuration.adoc["How to cook VIVIDUS?"].
+
+== Prerequisites
+
+=== Install and configure Git
+. Download and install http://git-scm.com/downloads[Git client]
```
Comment (ikalinin1) [inline docs/modules/ROOT/pages/getting-started.adoc:13]: It's not really required, isn't it? :)

**Review by ikalinin1 — APPROVED** [review]

File: docs/modules/ROOT/pages/getting-started.adoc (hunk 5 above)
Comment (valfirst) [reply docs/modules/ROOT/pages/getting-started.adoc:13]: that's true, I thought about this point, but decided to keep it as is in a sake of simplicity

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+. https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity[Configure] your identity
+
+=== Install Java
+. https://jdk.java.net/17/[LTS JDK 17] is the baseline. https://jdk.java.net/11/[LTS JDK 11] and the latest https://jdk.java.net/18/[JDK 18] are supported as well.
```
[the remark this answers is not in the document]
Comment (valfirst) [reply docs/modules/ROOT/pages/getting-started.adoc:17]: one more tool to manage in our docs, I would say it should be up to users

[the remark this answers is not in the document]
Comment (valfirst) [reply docs/modules/ROOT/pages/getting-started.adoc:17]: and check out windows installation: https://sdkman.io/install

**Review by uarlouski — APPROVED** [review]

**Pull request merged**